packages/devtools_app/lib/src/http/http_request_data.dart

```diff
@@ -553,18 +117,65 @@ class DartIOHttpRequestData extends HttpRequestData {
     };
   }
 
+  @override
+  String? get contentType {
+    if (responseHeaders == null || responseHeaders![_contentTypeKey] == null) {
```

Comment:
Create a local for `responseHeaders` to avoid the extra checks?

Reply:
done